modules/authentication/proc-enabling-authentication-with-rhsso.adoc

```diff
@@ -163,6 +168,20 @@ auth:
 ----
 --
 
+`auth.backstageTokenExpiration`::
```

Comment:
Note that setting this configuration does not have a impact on the originating performance and security issue, but imo would be still beneficial to include here so that users are aware of the existence of this config.